README.rst

```diff
@@ -2,7 +2,13 @@
 AOSOM-Streaming
 ###############
 
+**************
 Version v0.6.1
+**************
+
+Disclaimer
+==========
+WARNING: This application is for demo purposes only and should not be used in production.
 
 The Aosom Streaming project provides a packaged solution to collect
```

Comment:
Add an overview as heading at the same level as Disclaimer - right now it looks like all of this is in disclaimer.